Makefile

```diff
@@ -85,3 +90,15 @@ test: testopa
 .PHONY: testopa
 testopa:
 	docker run --rm -it -v ${PWD}/policies:/policies/ openpolicyagent/opa test -v /policies/
+
+.PHONY: package
+package:
+	mkdir -p karavi_authorization_${DOCKER_TAG}
+	cp ./deploy/rpm/x86_64/karavi-authorization-${VERSION_TAG}.x86_64.rpm karavi_authorization_${DOCKER_TAG}/
+	cp ./deploy/dist/microos-k3s-selinux.rpm karavi_authorization_${DOCKER_TAG}/
+	cp ./deploy/dist/centos7-k3s-selinux.rpm karavi_authorization_${DOCKER_TAG}/
+	cp ./deploy/dist/centos8-k3s-selinux.rpm karavi_authorization_${DOCKER_TAG}/
+	cp ./scripts/install_karavi_auth.sh karavi_authorization_${DOCKER_TAG}/
+	cp -r ./policies karavi_authorization_${DOCKER_TAG}/
+	tar -czvf package/karavi_authorization_${DOCKER_TAG}.tar.gz karavi_authorization_${DOCKER_TAG}
```

Comment:
Will the package/ directory be created automatically by tar command? If not need a `mkdir -p package` before this line.

Reply:
Done